airbyte-integrations/connectors/destination-s3/src/main/java/io/airbyte/integrations/destination/s3/S3DestinationConfig.java

```diff
@@ -135,6 +135,10 @@ public static S3DestinationConfig getS3DestinationConfig(final JsonNode config)
     return builder.get();
   }
 
+  private static String resolveBucketName(String originalBucketName) {
+    return originalBucketName.contains("/") ? originalBucketName.replaceAll("/","-") : originalBucketName;
```

Comment:
I guess that this validation is redundant, you can simply do `originalBucketName.replaceAll("/","-")` all the time

Reply:
> I guess that this validation is redundant, you can simply do `originalBucketName.replaceAll("/","-")` all the time

@alexandr-shegeda removed redundant validation